defaults/main.yml

```diff
@@ -1,18 +1,18 @@
 ---
-# defaults file for ansible-role-repo-openio-sds
+# defaults file for ansible-role-openio-repository
 
-openio_sds_release: '17.04'
+openio_repository_distribution: '17.04'
+openio_repository_state: present
 
-openio_sds_repo_user: ""
-openio_sds_repo_pass: ""
-openio_sds_repo_creds: "{% if openio_sds_repo_user and openio_sds_repo_pass %}{{ openio_sds_repo_user }}:{{ openio_sds_repo_pass }}@{% endif %}"
+# Credentials for private content
+openio_repository_user: ""
+openio_repository_pass: ""
+openio_repository_creds: "{% if openio_repository_user and openio_repository_pass %}{{ openio_repository_user }}:{{ openio_repository_pass }}@{% endif %}"
 
-openio_sds_repo_product: 'sds'
+openio_repository_product: 'sds'
 
-openio_sds_repo_baseurl: "http://{{ openio_sds_repo_creds }}mirror.openio.io/pub/repo/openio"
-openio_sds_repo_produrl: "{{ openio_sds_repo_baseurl }}/{{ openio_sds_repo_product }}"
-openio_sds_repo_releaseurl: "{{ openio_sds_repo_produrl }}/{{ openio_sds_release }}"
+openio_repository_manage_openstack_repository: true
+openio_repository_disable_policy_autostart: true
```

Comment:
This looks like deb-specific, maybe this should show in the variable name to avoid confusion.
what about `openio_repository_deb_disable_policy_autostart`, and if we have redhat specific ones we can use `openio_repository_rpm_` prefix...

Reply:
The long names does not bother me, but a description in the README seems sufficient

Reply:
OK, so what about putting it in a `defaults/Debian.yml` file instead ? In that case you can keep the name without the "_deb_"...

Reply:
it s works ? `defaults/Debian.yml`never seen that

Reply:
It looks like ansible will never support that, but you can do some manual processing, like :
https://github.com/sbitio/ansible-fail2ban/commit/70a6b594a3d8e6789119751f675cfd593b6f2a3e

I'll let you decide if you want to go down that path.